ArduCopter/GCS_Mavlink.cpp

```diff
@@ -1160,8 +1160,8 @@ void GCS_MAVLINK_Copter::handleMessage(const mavlink_message_t &msg)
         bool yaw_rate_ignore = packet.type_mask & MAVLINK_SET_POS_TYPE_MASK_YAW_RATE_IGNORE;
         bool force_set       = packet.type_mask & MAVLINK_SET_POS_TYPE_MASK_FORCE_SET;
 
-        // force_set true, do not accept command
-        if (force_set) {
+        // force_set is true when acceleration is to be commanded, do not accept command
```

Comment:
I think we need to be a little clearer with this comment. Something like:

// Force inputs are not supported
// Do not accpt command if force_set is true and acc_ignore is false

Reply:
Definitely. Fixed.